cosmo_tester/config_schemas/manager_image_names_rhel.yaml

```diff
@@ -1,11 +1,17 @@
 namespace: manager_image_names_rhel
-master:
+"master":
   description: Name of main manager image to test. This will be used in all manager tests.
-  default: cloudify-manager-premium-5.1.1-rhel
-installer:
+  default: cloudify-manager-premium-5.2.0.dev1-rhel
+"installer":
   description: Name of image with manager installer for bootstrap or cluster installs.
+  default: cfy-installer-premium-5.2.0.dev1-rhel
+"5_1_1_installer":
+  description: Name of image with 5.1.1 manager installer for bootstrap or cluster installs.
   default: cfy-installer-premium-5.1.1-rhel
-5_1_0_installer:
+"5_1_1":
+  description: Name of manager image for 5.1.1. All non master images are only used for upgrade (snapshot) tests.
```

Comment:
It is true that they are used only for "upgrade". But now with `cfy_manager upgrade`, this is not only snapshots. So "(snapshots)" needs to be deleted.

Reply:
Updated